server/game/cards/03_TWI/units/RuneHaakoSchemingSecond.ts

```diff
@@ -0,0 +1,42 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { StateWatcherRegistrar } from '../../../core/stateWatcher/StateWatcherRegistrar';
+import { CardsLeftPlayThisPhaseWatcher } from '../../../stateWatchers/CardsLeftPlayThisPhaseWatcher';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { WildcardCardType } from '../../../core/Constants';
+
+export default class RuneHaakoSchemingSecond extends NonLeaderUnitCard {
+    private cardsLeftPlayThisPhaseWatcher: CardsLeftPlayThisPhaseWatcher;
+
+    protected override getImplementationId() {
+        return {
+            id: '5333016146',
+            internalName: 'rune-haako#scheming-second',
+        };
+    }
+
+    protected override setupStateWatchers(registrar: StateWatcherRegistrar): void {
+        this.cardsLeftPlayThisPhaseWatcher = AbilityHelper.stateWatchers.cardsLeftPlayThisPhase(registrar, this);
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'If a friendly unit was defeated this phase, you may give a unit –1/–1 for this phase',
+            optional: true,
+            targetResolver: {
+                cardTypeFilter: WildcardCardType.Unit,
+                immediateEffect: AbilityHelper.immediateEffects.conditional({
```

Comment:
I'm wondering if the targetResolver should be inside the conditional, not the other way around. Veld can confirm

Reply:
@AMMayberry1

Reply:
I'm fine with it as-is, this does read slightly backwards but it functions the exact same since the target resolver will just auto-pass if the condition isn't met due to `noAction()` always saying it has no legal targets. Plus, if we inverted them we'd need to use `selectCard()`